_posts/2016-04-26-cloudready-linux-mint.markdown

```diff
@@ -0,0 +1,166 @@
+---
+layout: post
+title: Dual-Boot Cloudready and Linux Mint
+tags: linux-mint cloudready dual-boot grub
+username: likethemammal
+---
+
+![Cloudready Logo](https://p5.zdassets.com/hc/settings_assets/187734/200050403/SZGZzvDCxuqvTTwMdcXTrw-Screen_Shot_2015-09-18_at_9.30.01_AM.png)
+
+<center>*Source: Neverware*</center>
+
+## Prologue
+
+My girlfriend has been on an HP Chromebook 14 for while now. I was originally the one to recommend it and she's been generally satisfied expect for a few issues. One is that, even though there are many web based alternatives to common software, the Chromebook hardware isn't actually capable of running them for very long. Audio editing, flash gaming, and sometimes even regular usage causes scrolling, typing, and the like, to slug to a stutter.
+
+For a while I've wanted to setup one of my old machines for her to use. The machine would have to be as easy to use as Chrome OS, but capable of running traditional PC games and audio editing software (she's a musician).
+
+As I detailed in [my last post](), I recently got a Razer Blade gaming laptop setup as my personal machine. That freed up the laptop I had been using, an HP Probook 430 G1, to become hers.
+
+This post documents the process of setting up Cloudready and Linux Mint to dual boot on the same machine.
+
+
+#####What is Cloudready
+
+If you haven't heard of [Cloudready](http://www.neverware.com/), its software Neverware provides to easily install [Chromium OS](https://www.chromium.org/chromium-os) (the open-source version of Chrome OS) onto Windows machines.  Cloudready also handles automatically updating the OS, a feature not included normally in Chromium OS.
+
+Neverware's focus is on revamping schools' old computer labs. Since a lot of that hardware is still better than today's Chromebooks, it runs Chromium beautifully. They also have a setup process to preserve the current Windows install and "dual boot" that with Cloudready.
+
+I use quotes around "dual-boot" because, when running the dual-boot setup, Cloudready is actually installed **into** the Windows partition, instead of along side it, completely taking it over. Its own boot software then manages whether it boots the user's Windows install or Chromium.
+
+I've provided diagrams below to explain the difference between this and a traditional dual-boot setup. We're aiming for a traditional dual-boot setup with Cloudready and Linux Mint in parallel. We may even try for the triple-boot!
+
+
+![Linux Mint and Cloudready boot structures](http://i.imgur.com/mK8mdFu.png)
```

Comment:
you can host the image in this repo, and link to it using the relative path. I did that in one of my previous posts, I think the terrain generation one. Then we're not hotlinking to somewhere else for image hosting,